### PR TITLE
gh-127740: Add some more tests for earlier PR #127756

### PR DESCRIPTION
**Pull request by srinivasreddy**
gh-127740: Add some more tests for earlier PR #127756

via - https://github.com/python/cpython/pull/127756#discussion_r1879530712

### CODE REVIEW

**Comment by srinivasreddy** [comment]
cc @hauntsaninja @cfbolz

File: Lib/test/test_bytes.py [hunk 1]
```diff
@@ -464,6 +464,10 @@ def test_fromhex(self):
             with self.assertRaises(ValueError) as cm:
                 self.type2test.fromhex(value)
             self.assertIn("fromhex() arg must contain an even number of hexadecimal digits", str(cm.exception))
+        for value, position in (("a ", 1), (" aa a ", 5),(" aa a a ", 5)):
```
Comment (ZeroIntensity) [inline Lib/test/test_bytes.py]: Typo?
```suggestion
        for value, position in (("a ", 1), (" aa a ", 5), (" aa a a ", 5)):
```

**Review by hauntsaninja — APPROVED** [review]
Thank you!

**Review by hauntsaninja — APPROVED** [review]
Thank you!

**Pull request merged**